model_utils/fields.py

```diff
@@ -126,6 +126,33 @@ def deconstruct(self):
         return name, path, args, kwargs
 
 
+class PersistentMonitorField(MonitorField):
+    """
+    Like MonitorField, but once the timestamp for the first change is recorded
+    it is never set again
+    """
+    def __init__(self, *args, **kwargs):
+        kwargs.setdefault('default', now())
```

Comment:
You're absolutely right. I got confused between field and model declarations. If this were ever to happen a nullable field initialised to None would be the best approach